test/plot-schema.json

```diff
@@ -3988,7 +3989,8 @@
       "values": [
        "auto",
        "linear",
-       "array"
+       "array",
+       "sync"
```

Comment:
Do you want to make this work for `polar.angularaxis`? 
If no let's not not add it to the API. 
Please see https://github.com/plotly/plotly.js/pull/6356/files#r1025586215 on how to remove it.